src/main/resources/application-production.properties

```diff
@@ -2,7 +2,7 @@ spring.datasource.url=${JDBC_DATABASE_URL}
 spring.datasource.username=${JDBC_DATABASE_USERNAME}
 spring.datasource.password=${JDBC_DATABASE_PASSWORD}
 
-app.showSwaggerUILink=${SHOW_SWAGGER_UI_LINK:${env.SHOW_SWAGGER_UI_LINK:false}}
+app.showSwaggerUILink=${SHOW_SWAGGER_UI_LINK:${env.SHOW_SWAGGER_UI_LINK:true}}
```

Comment:
We would not want to do this in production! By modifying it code that's being merged into main, it would be mean "ALWAYS show the swagger link", which we do NOT want. 

Instead, if you want the swagger link to show up on a particular dokku deployment, just do this on dokku-02:

```
dokku config:set courses-dev-evaniacheng SHOW_SWAGGER_UI_LINK=true
```

That way, the setting only impacts *your* dokku deployment, not the default for every production dokku ever from now on!